src/engine-registry.ts

```diff
@@ -33,6 +34,7 @@ export function handleContractRegistered(event: ContractRegistered): void {
     IGenArt721CoreV3_Base_Template.create(coreAddress);
     OwnableGenArt721CoreV3Contract_Template.create(coreAddress);
     IERC721GenArt721CoreV3Contract_Template.create(coreAddress);
+    IGenArt721CoreContractV3_Engine_Flex_Template.create(coreAddress);
```

Comment:
We could conditionally create this flex data source template only in the case where the contract is of type flex. I don't have any super strong opinions here other than if we keep it as-is, probably worth a dev comment indicating why it is okay to create a flex template for non-flex contracts (i.e. the event handlers don't overlap)